core/src/main/java/org/apache/brooklyn/util/core/internal/ssh/sshj/SshjTool.java

```diff
@@ -575,9 +585,16 @@ public int execCommands(Map<String,?> props, List<String> commands, Map<String,?
         }
         
         // If async is set, then do it as execScript
-        Boolean execAsync = getOptionalVal(props, PROP_EXEC_ASYNC);
+        Boolean execAsync = getOptionalVal(props, PROP_EXEC_ASYNC, this.execAsync);
         if (Boolean.TRUE.equals(execAsync) && BrooklynFeatureEnablement.isEnabled(BrooklynFeatureEnablement.FEATURE_SSH_ASYNC_EXEC)) {
-            return execScriptAsyncAndPoll(props, commands, env);
+            if (!props.containsKey(PROP_EXEC_ASYNC_POLLING_TIMEOUT.getName())) {
+                return execScriptAsyncAndPoll(
+                        MutableMap.<String, Object>builder().putAll(props).put(PROP_EXEC_ASYNC_POLLING_TIMEOUT.getName(), this.execAsyncPollingTimeout).build(),
```

Comment:
I'd pass `props` as is here and change [this line](https://github.com/apache/brooklyn-server/blob/1a279401aaf68a6970f96950fe45e0e72581680b/core/src/main/java/org/apache/brooklyn/util/core/internal/ssh/sshj/SshjTool.java#L354) to use `SshjTool.this.execAsyncPollingTimeout` as the default. And fall back to `FIVE_MINUTES` if null.